pkg/digest/digest.go

```diff
@@ -34,7 +36,9 @@ func Sha256(value string) string {
 func Sha1(contents []string) string {
 	h := sha1.New()
 	for _, content := range contents {
-		io.WriteString(h, content)
+		if _, err := io.WriteString(h, content); err != nil {
+			fmt.Fprintf(os.Stderr, "failed to write string to io.Write: %v", err)
```

Comment:
@hwdef  I don' t think we need an error handling here.
If you really want to add it here, it is better to change the error description.
Maybe failed to calculate sha-256 hash?

Reply:
I think you are right.
I will close this PR.